bigquery/storage/managedwriter/managed_stream.go

```diff
@@ -184,6 +185,11 @@ func (ms *ManagedStream) getStream(arc *storagepb.BigQueryWrite_AppendRowsClient
 		return nil, nil, ms.err
 	}
 
+	// Previous activity on the stream indicated it is not healthy, so propagate that as a reconnect.
```

Comment:
What are you seeing here? I thought passing in a 11MB message should be a in stream failure from server point of view. Or maybe some stubby layer failure is there before this?

Reply:
Discussed out of band.  Internal issue 239050626 filed internally with more details.